src/client/src/orangehrmCorporateDirectoryPlugin/pages/CorporateDirectory.vue

```diff
@@ -223,53 +232,49 @@ export default {
     });
 
     return {
+      fetchData,
       scrollerRef,
       ...toRefs(state),
-      fetchData,
-      filters,
+      ...toRefs(responsiveState),
     };
   },
 
   computed: {
     isMobile() {
       return this.windowWidth < 600;
     },
-    hasCurrentIndex() {
+    isEmployeeSelected() {
       return this.currentIndex >= 0;
     },
     oxdGridClasses() {
       return {
         'orangehrm-container': true,
-        'orangehrm-container-min-display': this.hasCurrentIndex,
+        'orangehrm-container-min-display': this.isEmployeeSelected,
       };
     },
+    colSize() {
+      if (this.windowWidth >= 1920) {
+        return this.isEmployeeSelected ? 5 : 7;
```

Comment:
Won't it be return this.isEmployeeSelected ? 6 : 7;

Reply:
I added 5 because 6 looked a little cluttered. what do you think @RajithaKumara 
![php74_orangehrm5_web_index php_directory_viewDirectory (2)](https://user-images.githubusercontent.com/12880059/174765723-c14d1a35-2bf0-477e-8f10-5f69e3cc3ae9.png)
![php74_orangehrm5_web_index php_directory_viewDirectory (4)](https://user-images.githubusercontent.com/12880059/174766430-e81f2b89-62b6-4867-a2b9-430726c756ee.png)